File: Mona_py/Commands/friends.py
import nextcord

from Mona_py.logger import log
from Mona_py.json_opener import voicelinesJSON


async def friends(interaction: nextcord.Interaction, character_name: str = None):
    log("command", f"Slash command /friends {character_name} used in {interaction.guild.name}")

    if character_name.lower not in voicelinesJSON["friends"]:
        await interaction.response.send_message("Who is that? I—I don't think I ever met them before...")
        return

    await interaction.response.send_message(voicelinesJSON["friends"][character_name])

File: Mona_py/Commands/JS/friends.js
const Discord = require('discord.js');

const main = require('../index.js');
const db = main.db;
const voicelinesJSON = main.voicelinesJSON;

module.exports = {
	name: 'friends',
	description: 'What mona thinks about her friends',
	execute(message, args) {
		console.command(`${"!mona friends".magenta} in ${(message.guild.name).toString().magenta}`)
		if(!args[0]) return message.channel.send("do !mona friends (name of genshin character)");
		if (!voicelinesJSON["friends"][args[0]]) return message.channel.send("Who is that? I—I don't think I ever met them before...")
		message.channel.send(voicelinesJSON["friends"][args[0]]); 
	},
};